site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
 
-![](/images/dextr_example.gif)
+![](/images/image224.jpg)
 
-### Feature backpropagating refinement scheme (f-BRS)
+You can use the **Selected opacity** slider in
+the [Objects sidebar](/docs/manual/basics/objects-sidebar/#appearance)
+to change the opacity of the polygon.
 
-The model allows to get a mask for an object using positive points
-(should be left-clicked on the foreground), and negative points
-(should be right-clicked on the background, if necessary).
-It is recommended to run the model on GPU, if possible.
+For more information, see
+[Annotation with polygons](/docs/manual/advanced/annotation-with-polygons/) section.
 
-![](/images/fbrs_example.gif)
+**Deleting points**
 
-### High Resolution Net (HRNet)
+If you are not satisfied with the result, you can set additional points or remove points.
+<br>To remove a point, do the following:
 
-The model allows to get a mask for an object using positive points
-(should be left-clicked on the foreground), and negative points
-(should be right-clicked on the background, if necessary).
-It is recommended to run the model on GPU, if possible.
+1. With the cursor, hover over the point you want to delete.
+2. If the point can be deleted, it will enlarge and the cursor will turn into a cross.
+3. Left-click on the point.
 
-![](/images/hrnet_example.gif)
+### Interactors Models
 
-### Inside-Outside-Guidance
+<!--lint disable maximum-line-length-->
 
-The model uses a bounding box and inside/outside points to create a mask.
-First of all, you need to create a bounding box, wrapping the object.
-Then you need to use positive and negative points to say the model where is a foreground,
-and where is a background. Negative points are optional.
+| Model                                                     | Description                                                                                                                                                                                                                                                                                                                       | Example                        |
+| --------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------ |
+| Deep extreme <br>cut (DEXTR)                              | This is an optimized version of the original model, <br>introduced at the end of 2017. It uses the <br>information about extreme points of an object <br>to get its mask. The mask is then converted to a polygon. <br>For now this is the fastest interactor on the CPU.                                                         | ![](/images/dextr_example.gif) |
```

Comment:
Probably it will be good to add a link on the article and GitHub repo for each model. Users will be able to find more information if they need it. For example, for DEXR:
GitHub: https://github.com/scaelles/DEXTR-PyTorch
Site: https://cvlsegmentation.github.io/dextr
Article: https://arxiv.org/pdf/1711.09081.pdf